R/calcCapTarget.R

```diff
@@ -1,104 +1,87 @@
 #' Capacity targets from two sources
+#'
 #' @description The capacity targets (GW)  at regional level are produced from two different databases-
-#' UNFCCC_NDC database, an update of the Rogelj 2017 paper (see readme in inputdata), and REN21 Global Renewables report
-#' The UNFCCC_NDC capacity targets are further broken down to conditional and unconditional targets.
-#' @author Aman Malik, Oliver Richters
```

Comment:
Please bring back all the `@author` tags. They are useful to evaluate who to ask for help and if the function is rather new or legacy code.